Makefile

```diff
@@ -76,15 +76,13 @@ $(GOIMPORTS):
 build: format
 	go build -v ./...
 	cp wavefront/config/wavefront.yaml install/wavefront/templates/
-	sed -i '' -E 's/namespace: istio-system/namespace: {{ .Values.namespaces.istio }}/' install/wavefront/templates/wavefront.yaml
-	sed -i '' -E 's/- metric/- metric.{{ .Values.namespaces.adapter }}/' install/wavefront/templates/wavefront.yaml
 	@echo "Build was successful!"
 
 # Builds the docker image for the project
 # Usage: make docker-build
 .PHONY: docker-build
 docker-build: build
-	docker build -t vmware/wavefront-adapter-for-istio:latest .
+	docker build --network=host -t vmware/wavefront-adapter-for-istio:latest .
```

Comment:
Why do we need this?

Reply:
Docker is not able to pull the photon dependencies, so had to add this flag as suggested by photon team.

Reply:
can we capture these details as comments. This would be useful to any engineer working on this in future.

Reply:
Added as comment and reverted the change.

Reply:
Actually here i meant to capture why we need --network=host in the docker build command. Didn't meant to ask for reverting the change. As long as having --network=host works on all machines/OS's running this container we can have it.

Reply:
I guess this flag doesn't work on mac https://docs.docker.com/network/host/ hence reverted.

Reply:
For reference: we discussed this offline - 
The network specified in build is only for downloading packages that are necessary for building image. The network specified in run is for the containers itself. 

But we agreed that we can revisit this as a part of automating the build process for adapter.